packages/core/getBusyTimes.ts

```diff
@@ -83,9 +83,10 @@ export async function getBusyTimes(params: {
   const endTimeDate =
     rescheduleUid && duration ? dayjs(endTime).add(duration, "minute").toDate() : new Date(endTime);
 
+  // startTime is less than endTimeDate and endTime grater than startTimeDate
   const sharedQuery = {
-    startTime: { gte: startTimeDate },
-    endTime: { lte: endTimeDate },
+    startTime: { lte: endTimeDate },
```

Comment:
Watch out, this isn't the only place this happens, https://github.com/calcom/cal.com/blob/969411041b75833ad6625d341a9d0b7ea535de43/packages/trpc/server/routers/viewer/slots/util.ts#L330

But such a good spot 💯

Reply:
Thank you, fixed it there as well 🙏